### PR TITLE
Ensure commands always silent for MSVC

### PR DESCRIPTION
**Pull request by dra27**
If a command is routed through bash because it's too long, the fact it's meant to be piped to NUL is forgotten. Ensure the bash command is piped to >/dev/null (with the slightly tedious recovery of the script needed if the command fails).

#68 needs to be merged first, because it corrects a magic number in this PR

### CODE REVIEW

File: reloc.ml [hunk 1]
```diff
@@ -198,27 +198,35 @@ let new_cmdline () =
...
 
     if !verbose >= 1 then Printf.printf "(call with bash: %s)\n%!" fn;
-    if Sys.command (Printf.sprintf "bash %s" fn) <> 0 then
+    let invoke = Printf.sprintf "bash %s" fn in
+    if Sys.command invoke <> 0 then begin
+      if pipe_to_null then begin
+        let oc = open_out fn in
+        output_string oc cmd;
+        close_out oc;
+        ignore (Sys.command invoke)
```
Comment (alainfrisch) [inline reloc.ml:223]: What's the purpose of running the command again without the redirection?  We don't do that in the "direct" case (where bash is not used); and we might get errors (printed to stderr) twice.

Comment (dra27) [reply reloc.ml:223]: Yes we do - see deleted lines 220 and 221 below.

It's a good point, though - the treatment on the other branch is for MSVC where errors are also sent to stdout. I think the correct thing is probably to do `&>/dev/null` on this side and a belt-and-braces `2>&1` on the other?

Comment (dra27) [reply reloc.ml:223]: The whole lot exists to get rid of those pesky "foo.c" lines from MSVC...

**Comment by dra27** [comment]
Rebased and I think ready to merge?

**Pull request merged**